LeadKit/LeadKit/Extensions/UIImage/UIImage+Text.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UIImage+Text.swift
+//  LeadKit
+//
+//  Created by Николай Ашанин on 28.09.16.
+//  Copyright © 2016 Touch Instinct. All rights reserved.
+//
+
+import UIKit
+
+public extension UIImage {
+
+    /**
+     creates a text label image.
+
+     - parameter text: The text to use in the label.
+     - parameter font: The font (default: System font of size 18)
+     - parameter color: The text color (default: White)
+     - parameter backgroundColor: The background color (default:Gray).
+     - parameter size: Image size (default: 10x10)
+     - parameter offset: Center offset (default: 0x0)
+     */
+    public convenience init?(text: String,
+                             font: UIFont = UIFont.systemFontOfSize(18),
```

Comment:
font is very important and one of most unstable parameters. it shouldn't have default value